### PR TITLE
Finishing both the instructions and folder structure guide

### PR DESCRIPTION
**Pull request by sean-afshar**
Finalizing this new draft of the 2 guides so that we can package and push this thing out

### CODE REVIEW

File: docs/FOLDER_STRUCTURE.md [hunk 1]
```diff
@@ -1,11 +1,64 @@
...
-This package expects that your data has already been pose tracked and proofread with
-[SLEAP](https://sleap.ai), and organized according to the specifications below.
+1. **All tracking data must be proofread with [SLEAP](https://sleap.ai) and exported to an analysis.h5 file.**
 
-**It is very important to note that any file ending in an asterisk can be generated from running certain functions in the triangulation and calibration modules. Files and folders not ending in an asterisk must exist before running any functions from sleap-anipose, and must exist in the structure detailed below.** 
+2. **The user has completed taking calibration videos or images.**
+
+3. **User data is organized according to the notion of a *session***.
+
+A session is a folder that contians videos for a single continuous recording, i.e. **there is only 1 video per vamera view in a session folder.** Sessions have a unique internal structure to keep track of files which are specific to the entire recording and which files are specific to the camera view. The basic structure of a session is shown below, with pre-requisite files and folders ending in an asterisk.
```
Comment (talmo) [inline docs/FOLDER_STRUCTURE.md]: vamera -> camera

File: docs/FOLDER_STRUCTURE.md [hunk 2]
```diff
@@ -1,11 +1,64 @@
...
+        |   top_view_frame_1.jpg*
+        |           .
+        |           .
+        |           .
+        └── top_view_frame_137.jpg*
+```
+
+## Dissecting session structure
+
+Within each session folder there are a series of subfolders corresponding to each camera view. In the diagram above, alongside these view subfolders there is a collection of files derived from running functions from sleap-anipose. 'calibration.toml', 'calibration.metadata.h5', and 'reprojection_errors.png' can be created from running the [calibration function and its relevant ancilliary function](sleap_anipose/calibration.py) while 'points3d.h5' is created from running the [triangulation function](sleap_anpipose/triangulation.py). In addition, while these files are saved to the session directory, it is not absolutely necessary to save them there. However, it is highly recommended to save these files to their respective sessions.
```
Comment (talmo) [inline docs/FOLDER_STRUCTURE.md]: Use backticks (``) instead of single quotes (') for `calibration.toml`, etc

File: docs/FOLDER_STRUCTURE.md [hunk 3]
```diff
@@ -1,11 +1,64 @@
...
+
+## Dissecting session structure
+
+Within each session folder there are a series of subfolders corresponding to each camera view. In the diagram above, alongside these view subfolders there is a collection of files derived from running functions from sleap-anipose. 'calibration.toml', 'calibration.metadata.h5', and 'reprojection_errors.png' can be created from running the [calibration function and its relevant ancilliary function](sleap_anipose/calibration.py) while 'points3d.h5' is created from running the [triangulation function](sleap_anpipose/triangulation.py). In addition, while these files are saved to the session directory, it is not absolutely necessary to save them there. However, it is highly recommended to save these files to their respective sessions. 
+
+On the other hand, the view subfolders are absolutely mandatory and must exist in the session before running calibration or triangulation in sleap-anipose. Furthermore, within each view subfolder it is mandatory to have the following:
+
+1. **The video of the experiment taken from the corresponding camera view**
+
+2. **Your tracked and proofread data saved to 2 files, a .slp file and an analysis.h5 file**
```
Comment (talmo) [inline docs/FOLDER_STRUCTURE.md:49]: Is the .slp file really required?

**Review by talmo — CHANGES_REQUESTED** [review]
Minor fixes

File: docs/FOLDER_STRUCTURE.md (hunk 1 above)
Comment (sean-afshar) [reply docs/FOLDER_STRUCTURE.md]: fixed

File: docs/FOLDER_STRUCTURE.md (hunk 3 above)
Comment (sean-afshar) [reply docs/FOLDER_STRUCTURE.md:49]: not really. ill put a little disclaimer stating that it is highly recommended to keep the file there anyways

File: docs/FOLDER_STRUCTURE.md (hunk 2 above)
Comment (sean-afshar) [reply docs/FOLDER_STRUCTURE.md]: resolved

**Review by talmo — APPROVED** [review]

**Pull request merged**